cis_proc.py

```diff
@@ -176,22 +178,24 @@ def main(tar_file, bids_dir, config, sub, ses=None, work_dir=None, n_procs=1):
                 shutil.copyfile(op.join(scan_work_dir, 'bids', dset_file),
                                 op.join(bids_dir, dset_file))
 
-        new_participants_df = pd.read_csv(op.join(scan_work_dir, 'bids/participants.tsv'),
-                                          sep='\t')
-        new_participants_df = new_participants_df.T.drop_duplicates().T
-        orig_participants_df = pd.read_csv(op.join(bids_dir, 'participants.tsv'),
-                                           sep='\t')
-        orig_participants_df = orig_participants_df.T.drop_duplicates().T
+        new_participants_df = pd.read_csv(
```

Comment:
I reworked this with method chaining to remove the extra reassignment of the `df` I can add a comment for clarity if need be

Reply:
I have a feeling the double-transpose is unnecessary if we use drop_duplicates' arguments.